packers/zkp.go

```diff
@@ -237,3 +240,83 @@ func unmarshalPubSignals(obj circuits.PubSignalsUnmarshaller, pubSignals []strin
 func (p *ZKPPacker) MediaType() iden3comm.MediaType {
 	return MediaTypeZKPMessage
 }
+
+// DefaultZKPPakcerOption is a function that sets the default ZKP packer options
+type DefaultZKPPakcerOption func(*defaultZKPPacker)
+
+// WithAuthVerifyDelay sets the delay for the auth verification
+func WithAuthVerifyDelay(delay time.Duration) DefaultZKPPakcerOption {
+	return func(p *defaultZKPPacker) {
+		p.authVerifyDelay = delay
+	}
+}
+
+type defaultZKPPacker struct {
+	resolvers       map[int]eth.Resolver
+	authVerifyDelay time.Duration
+}
+
+// DefaultZKPPacker creates a default ZKP packer with the provided verification key and resolvers
+func DefaultZKPPacker(verificationKey []byte, resolvers map[int]eth.Resolver, opts ...DefaultZKPPakcerOption) *ZKPPacker {
```

Comment:
from the code you are right, it gives only unpack functionality. So I do not think it should be named `DefaultZKPPacker`.  ZKPUnpacker is ok name maybe?